stdlib/2/popen2.pyi

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Iterable, List, Optional, Union, Tuple, TypeVar
+
+_T = TypeVar('_T')
+
+__all__ = ...  # type: List[str]
+
+class Popen3:
+    sts = ...  # type: int
+    cmd = ...  # type: Iterable
+    pid = ...  # type: int
+    tochild = ...  # type: file
+    fromchild = ...  # type: file
+    childerr = ...  # type: Optional[file]
+    def __init__(self, cmd: Iterable = ..., capturestderr: bool = ..., bufsize: int = ...) -> None: ...
```

Comment:
The source code for `popen2.py` seems to indicate that `cmd` can be a string, as well:

> On UNIX, 'cmd' may be a sequence, in which case arguments will be passed directly to the program without shell intervention (as with os.spawnv()).  If 'cmd' is a string it will be passed to the shell (as with os.system()).